From 5d1830516c6a0cd5c2ea248f33f85bf67382555c Mon Sep 17 00:00:00 2001
From: liushuyu <liushuyu011@gmail.com>
Date: Mon, 13 Oct 2025 11:02:29 +0800
Subject: [PATCH 02/15] loongvpu: include/hantro_device.h: defined MAX macro
 only when the

upstream kernel headers do not define this macro
---
 include/hantro_device.h | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/include/hantro_device.h b/include/hantro_device.h
index e610769..4058851 100644
--- a/include/hantro_device.h
+++ b/include/hantro_device.h
@@ -23,7 +23,9 @@
 
 #include "hantro.h"
 
+#ifndef MAX
 #define MAX(a, b) (((a) > (b)) ? (a) : (b))
+#endif
 
 /* #define MAX(a, b)                 \
  * ({ typeof(a) a_ = (a);            \
-- 
2.54.0

